src/utils/extensions/telemetry/telemetry.ts

```diff
@@ -0,0 +1,106 @@
+import { kubevirtConsole as kvConsole } from '@kubevirt-utils/utils/utils';
+
+const SEGMENT_KEY =
```

Comment:
Am I guessing all these files are from the Console? or some kind of demo?

Reply:
Yes, this is how the API key is retrieved in Console.